systemd/README.md

```diff
@@ -1,69 +1,5 @@
-Refer to documentation [Running witnet-rust from source code](https://github.com/witnet/witnet-rust/blob/master/docs/get-started/installation/from-source.md)
+Refer to documentation [Run as systemd service](https://github.com/witnet/witnet-rust/blob/master/docs/get-started/installation/from-source.md#run-as-systemd-service) to read the steps to create systemd service for witnet node.
 
-The steps to create systemd service for witnet node (all the steps with `root`).
+Run runner.sh to download and install automatically witnet node latest version. It can also be used to update it to the latest version because it will keep the hiden directory and the configuration file.
```

Comment:
```suggestion
Run `runner.sh` to automatically download and install the latest version of the Witnet node software (`witnet-rust`). It can also be used to update it to the latest version because it will keep all the block chain data and the configuration file in the hidden `.witnet` directory.
```